test/unit/ObjectMakrTests/processPropertyArray.js

```diff
@@ -0,0 +1,31 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("leaves Object types unchanged", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            var properties = mocks.mockPropertyArray();
+
+            //Act
+            properties.Starfish = objectMaker.processPropertyArray(properties.Starfish);
+
+            // Assert
+            expect(typeof properties.Creature).to.deep.equal("object");
```

Comment:
I wanted to test that the `Creature` was stil left as an array (which I'm not sure is necessary) and that `Starfish` was changed into an object, but forgot that an array is also an object.
If this is something fine to test for, can I use `Array.isArray`?
Otherwise I'm not sure what else to test for in this.